docs/notary-server.md

```diff
@@ -0,0 +1,108 @@
+<!--[metadata]>
++++
+title = "Notary CLI"
+description = "Description of the Notary CLI"
+keywords = ["docker, notary, trust, image, signing, repository, cli"]
+[menu.main]
+parent="mn_notary"
++++
+<![end-metadata]-->
+
+# Notary Server
+
+The notary repository comes with Dockerfiles and a docker-compose file
+to facilitate development. Simply run the following commands to start
+a notary server with a temporary MySQL database in containers:
+
+```
+$ docker-compose build
+$ docker-compose up
+```
+
+If you are on Mac OSX with boot2docker or kitematic, you'll need to
+update your hosts file such that the name `notary` is associated with
```

Comment:
"that the name `notary-server` is associated with the IP address...`127.0.0.1 notary-server` to your hosts file."

Since the client is still looking for the hostname notary-server, right?